Check/Content/ContentTypes.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\Content\ContentTypes.
+ */
+
+/**
+ * Class SiteAuditCheckContentContentTypes.
+ */
+class SiteAuditCheckContentContentTypes extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Content entity types');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Available content entity types and counts');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {}
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {
+    $ret_val = '';
+
+    if (empty($this->registry['content_type_counts'])) {
+      if (drush_get_option('detail')) {
+        return dt('No nodes exist.');
+      }
+      return $ret_val;
+    }
+
+    $ret_val .= "Total: {$this->registry['node_count']} nodes";
+    if (drush_get_option('html') == TRUE) {
+      $ret_val = "<p>$ret_val</p>";
+    }
+    else {
+      $ret_val .= PHP_EOL;
+    }
+
+    if (drush_get_option('html') == TRUE) {
+      $ret_val .= '<table class="table table-condensed">';
+      $ret_val .= "<thead><tr><th>Content Type</th><th>Node Count</th></tr></thead>";
```

Comment:
Wrap text (not HTML) in `dt`. If that's not like that in the v7 version, let me know.